Readme.txt

```diff
@@ -1,77 +1,138 @@
                                 MORSE RUNNER
                               Contest Simulator
                                   freeware
-
-                   Version 1.80 Field Day Contest Prototype
+                                  
+                        Version 1.82 - ARRL DX Contest
+            The third release of the Morse Runner Community Edition
 
                Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                       http://www.dxatlas.com/MorseRunner/
 
-        Copyright (C) 2022 Morse Runner Community Edition Contributors
+        Copyright (C) 2022-2023 Morse Runner Community Edition Contributors
                    https://www.github.com/w7sst/MorseRunner/
 
 
 JOIN OUR COMMUNITY
   You are invited to join our community effort.
   For more information on the Morse Runner Community Edition project,
-  please visit https://github.com/w7sst/MorseRunner#readme.
+  please visit https://github.com/w7sst/MorseRunner#readme.  
+  Feedback can be left in our discussions area. 
 
+  You can also discuss or leave feedback at [MorseRunnerCE on groups.io](https://groups.io/g/MorseRunnerCE).
+  
 PLATFORMS
 
   - Windows XP/7/8/10/11
   - works on Linux systems under WINE (info TNX F8BQQ).
 
 INSTALLATION
-
-  - Uncompress the file to any folder and run "MorseRunner.exe"
+  Open the zip file, extract the folder to your desktop, and run `MorseRunner.exe` in that folder.
 
 UNINSTALLATION
 
   - Delete MorseRunner directory.
 
 CONFIGURATION
 
+  First time setup instructions can be found within the program under help. 
+
   Contest Selection
     1) Select the desired contest using the Contest drop-down list.
     2) Enter the Contest Exchange in the Exchange field;
-       error messages will be displayed in the status area.
+       any error messages will be displayed in the status area.
+       * Cut Numbers in sending exhanges coming soon.
+
+  Contest Info
```

Comment:
This seems like a lot of extra detail to have this early in the document. Perhaps we should move this to the end as an Appendix?

Reply:
Agreed. I tried putting it later but struggled with it. Even debated even adding it. But it's good info here until we get it in the help menu. 

Appendix solves this for me. :)

Reply:
Or we can put this as a separate document or into a future `Help` file and not the release notes.

Reply:
Yes that is precisely it. Our readme is currently the help file. There is an issue assigned to me to write a separate help file. It should be ready for the next release. 

So this readme will point to a help file in the program for those looking for it in the future.

Reply:
I think I understand now - yes, this `readme.txt` also serves as the Help file. The future readme file will probably be more than a simple text file, perhaps using Windows Help file format, or even a PDF. It would be nice to find a tool for building a help file. We also want a Help file design that will lead to future localization. We want to pick something that be integrated into MRCE to allow F1 key to work (when not in Run Mode). So much to consider.

Reply:
I quoted the above in [Help File needs to be written](https://github.com/w7sst/MorseRunner/issues/67#top)
#67 for reference. 

Should I resolve?

Reply:
I marked resolved for you.